docs/data.md

```diff
@@ -151,3 +151,128 @@ i.e., push the computation region settings high in the code structure.
 However, for running a job, the computational region should be somewhere
 otherwise it will rely on whatever is the (last set) computational region in the
 current mapset.
+
+## Using local scratch space for temporary data
+
+### Running a single job
+If you are processing a large number of temporary files with GRASS GIS, it can be useful to write the   
+temporary files to local scratch space on the compute node (can decrease run time). In your  
+[job submission script](https://github.com/ncsu-geoforall-lab/grass-gis-on-hpc-henry2/blob/main/docs/jobs.md#running-a-single-job-calling-a-grass-module) you will need to add the following code, e.g.:  
+
+```tcsh
+#!/bin/tcsh
+#BSUB -n 1
+#BSUB -W 48:00
+#BSUB -R span[hosts=1]
+#BSUB -oo comp1_out
+#BSUB -eo comp1_err
+#BSUB -J comp1
+
+#########   LOAD MODULES   #########
+module use --append /usr/local/usrapps/geospatial/modulefiles/
+module load grass
+
+#########   CODE TO WRITE TO LOCAL SCRATCH   #########
+# Set the path for the temporary file directory
+# $LSB_JOBID will reference the job ID of the submission
+setenv TMPDIR /scratch/$LSB_JOBID
+
+# Create the directory 
+mkdir -p /scratch/$LSB_JOBID/grassdata/albers
+
+# Link the PERMANENT mapset of interest to the temporary file directory using a symlink
+ln -s /rsstu/users/r/rkmeente/FUTURES/grassdata/albers/PERMANENT/ /scratch/$LSB_JOBID/grassdata/albers/
+
+# The resulting directory tree should look something like this:
+tree /scratch/$LSB_JOBID/grassdata/
+/scratch/65834/grassdata/
+└── albers
+    └── PERMANENT -> /rsstu/users/r/rkmeente/FUTURES/grassdata/albers/PERMANENT/
```

Comment:
Having a "debugging" command and its output may be confusing. Can you just add that at the end noting that _tree_ would have to be executed in the "above script"?

Reply:
You're right, I've modified the document according to your comment in both locations where inspecting the tree is stated (i.e., Running a single job and Running a parallel job on multiple nodes using pynodelauncher). Please let me know  if you think it needs further refinement.

Reply:
Great, that works. It is good to keep the script minimal I think. Just see my other comment about the _tree_ command.